server/server/src/central.rs

```diff
@@ -0,0 +1,57 @@
+use actix_web::{
+    http::header::HeaderValue,
+    post,
+    web::{self, Data, Json},
+    HttpRequest, HttpResponse,
+};
+use base64::{prelude::BASE64_STANDARD, Engine};
+
+use crate::central_server_only;
+use service::{
+    apis::api_on_central::{self, CentralApiError, NameStoreJoinParams, SiteAuth},
+    service_provider::ServiceProvider,
+};
+
+pub fn config_central(cfg: &mut web::ServiceConfig) {
+    cfg.service(
+        web::scope("central")
+            .wrap(central_server_only())
+            .service(patient_name_store_join),
```

Comment:
Keen for thoughts here, added a new API `central` - for APIs exposed only from OMS central, but aren't sync related (I think we only had `central/sync` before)

@andreievg you happy with this?

Reply:
Yeah, but a slight confusion, i would structure folder then

central
-- mod.rs -> here we map to 'central' and add sync/name_store_join
-- sync.rs
-- name_store_join.rs